awx/ui/client/src/management-jobs/card/card.controller.js

```diff
@@ -45,69 +45,91 @@ export default
                     });
                 };
 
+                var launchManagementJob = function (defaultUrl){
+                  var data = {};
+                  Rest.setUrl(defaultUrl);
+                  Rest.post(data)
```

Comment:
I don't think you need to pass `data` for this type of launch. Also, this var isn't defined in the scope of this function.

Reply:
update: oh there it is